test/services/data_model_engine.js

```diff
@@ -1152,9 +1159,43 @@ describe('Data Model Engine', () => {
     });
 
     it('does not store bundle if validation failed', async () => {
-      mockEntityBuilder.validateBundle.rejects();
+      mockEntityBuilder.validateBundle.throws();
       await expect(modelEngine.downloadBundle(bundleId, sheltererId)).to.be.rejected;
       expect(mockEntityRepository.storeBundle).to.be.not.called;
     });
+
+    it('does not store bundle if on-chain verification failed', async () => {
+      verifyBundleStub.rejects();
+      await expect(modelEngine.downloadBundle(bundleId, sheltererId)).to.be.rejected;
+      expect(mockEntityRepository.storeBundle).to.be.not.called;
+    });
+  });
+
+  describe('Verifying a bundle', async () => {
+    const downloadedBundle = createBundle({},['first', 'second', 'third']);
```

Comment:
Missed space after `,`. Should check why linter did not fail

Reply:
👍